WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -127,30 +130,23 @@ class NotificationsViewController: UITableViewController {
         tableViewHandler.updateRowAnimation = .none
     }
 
-    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
-        super.viewWillTransition(to: size, with: coordinator)
-        // Note: We're assuming `tableViewHandler` might be nil. Weird case in which the view
-        // hasn't loaded, yet, but the method is still executed.
-        tableViewHandler?.clearCachedRowHeights()
-    }
-
 
     // MARK: - UITableView Methods
 
-    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
-        return nil
+    override func tableView(_ tableView: UITableView, estimatedHeightForHeaderInSection section: Int) -> CGFloat {
```

Comment:
Yeah, I'm not actually sure why I suggested that, when you're computing a value there... Refactoring to reuse the caching code is a good idea. I was going to suggest it, but didn't know if it was something you'd want to tackle in this PR.